testsuite/MDAnalysisTests/parallelism/test_multiprocessing.py

```diff
@@ -0,0 +1,209 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import multiprocessing
+
+import numpy as np
+import pytest
+import pickle
+from numpy.testing import assert_equal
+
+import MDAnalysis as mda
+from MDAnalysis.lib.picklable_file_io import pickle_open
+from MDAnalysis.coordinates.core import get_reader_for
+
+from MDAnalysisTests.datafiles import (
+    CRD,
+    PSF, DCD,
+    DMS,
+    DLP_CONFIG,
+    DLP_HISTORY,
+    INPCRD,
+    GMS_ASYMOPT,
+    GMS_SYMOPT,
+    GRO,
+    GSD,
+    GSD_long,
+    LAMMPSdata_mini,
+    LAMMPSDUMP,
+    mol2_molecules,
+    MMTF,
+    NCDF,
+    PDB, PDB_small, PDB_multiframe,
+    PDBQT_input,
+    PQR,
+    TRR,
+    TRJ,
+    TRZ,
+    TXYZ,
+    XTC,
+    XPDB_small,
+    XYZ_mini, XYZ, XYZ_bz2,
+)
+
+
+def textio_line(file, i):
+    return file.readlines()[i]
+
+
+def test_multiprocess_fileio():
```

Comment:
you already test that files can be pickled that is enough as multiprocessing is using pickles internally.